test/unit/logic/transaction.js

```diff
@@ -194,15 +194,12 @@ describe('transaction', () => {
 
 		it('should throw an error Argument must be a valid hex string.', done => {
 			const inValidTransaction = Object.assign({}, validTransaction);
-			let err = null;
 			inValidTransaction.senderPublicKey =
 				'c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6fx';
 			try {
-				expect(transactionLogic.sign(keyPair, inValidTransaction));
+				transactionLogic.sign(keyPair, inValidTransaction);
 			} catch (e) {
-				err = e.message;
-			} finally {
-				expect(err).to.equal('Argument must be a valid hex string.');
+				expect(e.message).to.equal('Argument must be a valid hex string.');
```

Comment:
Change of behavior, previously we were always expecting error, now we only expect error if it goes into `catch` block, is this correct?

Reply:
It can lead to false-positive. Previously `expect` was in `finally` block, so it was always executed. Now the expectation will be checked only if there will be an error caught in `catch` block. If there is no error - the test will pass.
Maybe we should use something like `expect(transactionLogic.sign(keyPair, inValidTransaction)).to.throw(...);` and remove `try...catch` block?